.editorconfig

```diff
@@ -0,0 +1,9 @@
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.js]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
No final new line 🤣

Reply:
haha.  Yeah, I added standard linting to keep whitespace PRs to a minimum.